docs/source/start.rst

```diff
@@ -253,13 +253,13 @@ an entangled state between qubits 0 and 1 (that's what the "CNOT" gate does). Fi
 .. code:: python
```

Comment:
Why is this file getting reverted?

Reply:
oh weird, didn't mean to do that. Is there a difference?

Reply:
I think this was a result of fetching master midway through

Reply:
(The diff looks correct)

Reply:
what do you mean the diff looks correct? The lines you're removing are the lines I changed when I updated this file. Can you please remove these changes?

Reply:
@lcapelluto not sure whether this is an issue still, tried fixing it